Xwt.Mac/Xwt.Mac/TableViewBackend.cs

```diff
@@ -41,6 +42,13 @@ public abstract class TableViewBackend<T,S>: ViewBackend<NSScrollView,S>, ITable
 		ScrollView scroll;
 		NSObject selChangeObserver;
 		NormalClipView clipView;
+
+		Dictionary<CellView,CellInfo> cellViews = new Dictionary<CellView, CellInfo> ();
+
+		class CellInfo {
+			public ICellRenderer Cell;
+			public int Column;
+		}
```

Comment:
I've removed the Mapping in ec2daf9a2943fa62a162c703ada4238a759f6646. Now the column is retrieved directly from the CellViewBackend.